src/sustainable-web-design.test.js

```diff
@@ -31,6 +31,12 @@ describe("sustainable web design model", () => {
     });
   });
 
+  describe("perVisit", () => {
+    it("should return a single number for CO2 emissions", () => {
+      expect(typeof swd.perVisit(2257715.2)).toBe("number");
```

Comment:
I think this test demonstrates the intent, that we want a simple number back.

However 2257715.2 doesn't really do us many favours in terms knowing how this would be different to the number returned by the `perbyte` figure. 

Can we discuss this tomorrow to work out a sensible approach for looking at numbers, based on what we learned sanity checking out numbers for the spreadsheet work?

Reply:
We've agreed we'll address this in a separate issue.